### PR TITLE
Force deny writing if disk quota exceeded

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

https://github.com/milvus-io/milvus/issues/18635

### CODE REVIEW

**Comment by bigsheeper** [comment]
rerun ut

File: internal/datacoord/meta.go [hunk 1]
```diff
@@ -196,6 +196,18 @@ func (m *meta) GetNumRowsOfCollection(collectionID UniqueID) int64 {
 	return ret
 }
 
+// GetTotalBinlogSize returns the total size of all segments in cluster.
+func (m *meta) GetTotalBinlogSize() int64 {
```
Comment (czs007) [inline internal/datacoord/meta.go:200]: Please specify the unit(byte? KB? MB?) in the code comments.

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Pull request merged**